native/c/zowex.cpp

```diff
@@ -852,8 +852,8 @@ int handle_console_issue(ZCLIResult result)
   int rc = 0;
   ZCN zcn = {0};
 
-  string console_name(result.get_option("--console-name").get_value());
-  string command(result.get_positional("command").get_value());
+  string console_name(result.get_option("--console-name")->get_value());
+  string command(result.get_positional("command")->get_value());
```

Comment:
There are a few missed statements in the above function 😋 
`handle_job_cancel`

```cpp
  string jobid(result.get_positional("jobid").get_value());
  string option_dump(result.get_option("--dump").get_value());
  string option_force(result.get_option("--force").get_value());
  string option_purge(result.get_option("--purge").get_value());
  string option_restart(result.get_option("--restart").get_value());
```

Reply:
fixt